README.rst

```diff
@@ -129,8 +129,8 @@ page or ask us at our active Gitter channel, where we have maintainers from
 all over the world. We appreciate any help!
 
 We also have a
-`newcomer guide <http://coala.io/newcomer>`__
-to help you get started by fixing an issue yourself! If you get stuck anywhere
+`Newcomers Guide <http://coala.io/newcomer>`__ page
```

Comment:
`Newcomer's Guide` would be better here.
Usage of `page` doesn't really make sense, no?

Reply:
Also use `https`, instead of `http` :)

Reply:
To maintain consistency it should be `Newcomers' Guide` instead of `Newcomer's Guide`

Reply:
Yes, apostrophe after s.